src/MICore/PlatformUtilities.cs

```diff
@@ -131,6 +131,33 @@ public static string GetEnvironmentVariable(this ProcessStartInfo processStartIn
 #endif
             return null;
         }
+
+        public static string UnixPathToWindowsPath(string unixPath)
+        {
+            return unixPath.Replace('/', '\\');
+        }
+
+        public static string WindowsPathToUnixPath(string windowsPath)
+        {
+            return windowsPath.Replace('\\', '/');
```

Comment:
this might not work properly if they are using network shares.

Reply:
I think this might be broken if there are mounted paths in Windows and you are locally on a unix machine. 

E.g. the Windows network shares will have a drive `Z:\` or UNC path `\\path\to\UNC` but then you will need to map it to a Unix mnt path. E.g. `/mnt/UNCPath`